google/cloud/bigquery/_job_helpers.py

```diff
@@ -123,7 +147,13 @@ def do_query():
     return future
 
 
-def _to_query_request(job_config: Optional[job.QueryJobConfig]) -> Dict[str, Any]:
+def _to_query_request(
```

Comment:
If destination table is specified, we can't do `jobs.query`, need to fallback `jobs.insert`.